src/main/java/net/coderbot/iris/UpdateChecker.java

```diff
@@ -113,8 +129,12 @@ public Component getUpdateMessage() {
 	}
 
 	public String getUpdateLink() {
```

Comment:
We're gonna need some `@Nullable` annotations on some of these methods... Or just return Optional